python/cugraph/tests/test_connectivity.py

```diff
@@ -205,3 +190,45 @@ def test_strong_cc(graph_file):
 
     cg_vertices = sorted(lst_cg_components[idx])
     assert nx_vertices == cg_vertices
+
+
+# Test all combinations of default/managed and pooled/non-pooled allocation
```

Comment:
Is this a comment a FIXME? If not, then I don't think that is happening.

Reply:
fixed